input/pagecontent/variance.md

```diff
@@ -1,10 +1,12 @@
-AU Realm FHIR implementation guide projects are required to follow specific publishing guidelines:
+This version of the AU Core FHIR IG complies with Version 1.0 of the [HL7 Australia FHIR Work Group AU Variance Requirements](https://confluence.hl7.org/display/HA/Process%3A++AU+FHIR+IG+Variance+Requirements?preview=/239212390/281217432/HL7%20AU%20Variance%20Requirements%20v1.0.pdf).
```

Comment:
Missing the change to specify the exact version of the Variance Requirements doc in the hyperlink text.

Reply:
Version specified in text and link is to v1.0 pdf document. Please let me know if I have misunderstood your comment.

Reply:
As discussed in IG triage - hyperlinked 'Version 1.0 of the HL7 Australia FHIR Work Group AU Variance Requirements' to satisfy ballot resolution.